src/main/resources/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep/help-scanner_en.properties

```diff
@@ -1,5 +1,5 @@
 #NLS_MESSAGEFORMAT_ALL 
 #NLS_ENCODING=UTF8 
 
-description.test.type=The type of test to run.
+description.test.type=The type of test to run. Note: Dynamic Scan technology support is only for Cloud users.
 
```

Comment:
Let's not change this message for now. I would prefer to use the field validation to give the user an error in the Jenkins UI if they choose DAST when pointing to a local service.